go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/open-policy-agent/cert-controller v0.10.1
 	github.com/prometheus/client_golang v1.18.0
 	github.com/prometheus/client_model v0.5.0
-	github.com/ray-project/kuberay/ray-operator v1.0.0
+	github.com/ray-project/kuberay/ray-operator v0.0.0-20240102030253-af4f6ac85d63
```

Comment:
FYI @alculquicondor @vicentefb this is still pointing to master. Last time I spoke with @kevin85421 the next release won't be until ~Feb, are we okay merging latest version of the module for now and updating this to v1.1 or should we block on it?

Reply:
I'm ok moving forward like this, but we can probably fix the dependency on a path release, once ray-operator is released.